src/vehicle.h

```diff
@@ -270,6 +270,8 @@ struct vehicle_part : public JsonSerializer, public JsonDeserializer
     /** Get part definition common to all parts of this type */
     const vpart_info &info() const;
 
+    virtual item get_base() const;
```

Comment:
Why is this `virtual`? The class isn't intended to have virtual functions, it hasn't any other virtual functions at all.

And it should probably return a const reference instead of a copy.